### ISSUE TITLE
Version 5.0: URL in the href attribute of the sjt:tree tag is escaped

### ISSUE THREAD

**Issue by amellie** [struts-community-plugins/struts2-jquery#315]
Hi all,

I have the following code working before the upgrade to 5.0.0, however, the URL ``myUrl`` is escaped in 5.0.0. Instead of `/data/myAction.html?id1=xx&id2=yy`, it is trying to make a request to `/data/myAction.html?id1=xx&amp;id2=yy` (with the `&amp;`) which is incorrect

```
<s:url var="myUrl" action="myAction" namespace="/data" escapeAmp="false">
    <s:param name="id1" value="%{id1}"/>
    <s:param name="id2" value="%{id2}"/>
</s:url>

<sjt:tree name="myTree" 
               href="%{myUrl}" 
               jstreetheme="default" jstreethemeVariant="small" 
               checkbox="true" openAllOnLoad="false"/>
```

Any ideas how to get this working?

**Comment by lukaszlenart** [comment]
Looks the same as #311 which was already addressed

**Comment by amellie** [comment]
I can confirm that this is fixed in 5.0.1
